node/src/types/chainspec/protocol_config.rs

```diff
@@ -36,12 +42,38 @@ impl Display for ActivationPoint {
     }
 }
 
+/// Type storing the information about modifications to be applied to the global state.
+/// It stores the serialized `StoredValue`s corresponding to keys to be modified.
+#[derive(Clone, PartialEq, Eq, Serialize, Deserialize, DataSize, Debug)]
+pub struct GlobalStateUpdate(pub(crate) BTreeMap<Key, Vec<u8>>);
+
+impl ToBytes for GlobalStateUpdate {
+    fn to_bytes(&self) -> Result<Vec<u8>, bytesrepr::Error> {
+        self.0.to_bytes()
+    }
+
+    fn serialized_length(&self) -> usize {
+        self.0.serialized_length()
+    }
+}
+
+impl FromBytes for GlobalStateUpdate {
+    fn from_bytes(bytes: &[u8]) -> Result<(Self, &[u8]), bytesrepr::Error> {
+        let (update, remainder) = BTreeMap::<Key, Vec<u8>>::from_bytes(bytes)?;
+        let global_state_update = GlobalStateUpdate(update);
+        Ok((global_state_update, remainder))
+    }
+}
+
 #[derive(Clone, PartialEq, Eq, Serialize, Deserialize, DataSize, Debug)]
 pub struct ProtocolConfig {
     #[data_size(skip)]
     pub(crate) version: Version,
     /// This protocol config applies to the era begun immediately after the activation point.
     pub(crate) activation_point: ActivationPoint,
+    /// Any arbitrary updates we might want to make to the global state at the start of the first
+    /// era after the activation point.
```

Comment:
```suggestion
    /// era specified in the activation point.
```

The activation point had its meaning changed to represent the era ID of the era at which this protocol version starts.

I just noticed that the comment for the `activation_point` member is outdated too - would you mind updating it?